### PR TITLE
Pam 441 multiple thens

### PR DESCRIPTION
**Pull request by pamgoodrich**
Added doc for:

- Multiple Thens
- Import custom functions
- JSON schema editor in function options

### CODE REVIEW

File: 2a.-style-guides/a.style-guide-projects.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Stoplight style guides add consistency, reusability, and easy governance to your
 - Verify that every endpoint has security settings
 - Ensure URL standards are set and followed
 
-Stoplight uses [Spectral](https://meta.stoplight.io/docs/spectral/ZG9jOjYx-overview) to validate your API designs based on enabled style guides. You'll receive immediate feedback in the **Validation List** as you work in Studio.
+Stoplight uses [Spectral](https://meta.stoplight.io/docs/spectral/ZG9jOjYx-overview) to validate your API designs based on enabled style guides. You'll receive immediate feedback in the **API Design Checker** as you work. Select each entry in the list to go directly to the **Code** view for that problem. 
 
-![style-guide-feedback2.png](https://meta.stoplight.io/api/v1/projects/cHJqOjI/images/GqiaANFtj80)
+![API Designer Checker Feedback](https://stoplight.io/api/v1/projects/cHJqOjI/images/n1wKmilw5uk)
```
Comment (dotslashderek) [inline 2a.-style-guides/a.style-guide-projects.md:15]: Might not matter, just noticed it's called "API Design Checker" above and "API Designer Checker" here - realize it's just alt-text for an image, so probably doesn't matter

File: 2a.-style-guides/c-create-rules.md [hunk 2]
```diff
@@ -11,72 +11,55 @@ See these [rule examples](g-rule-examples.md) to get started.
...
 
-- **Property**
+### 3. Then Settings
 
-  Add properties to reference a specific entity in the target. For example, when you target the `API_Tags` area of the OAS 3.1 specification, enter `description` to enforce rules on tag descriptions.
+Use **Then** settings to further refine the rule's scope (optional) and select a function that determines how design content will be evaluated. For example, the core function `truthy` checks for a value that's not `false`, `""`, `0`, `null`, or `undefined`.
 
-![Add property to target](https://stoplight.io/api/v1/projects/cHJqOjI/images/bZce0HHfE1s)
+You can create multiple **Then** settings per rule. This enables you to applly multiple functions to a rule rather than create a rule for each function you need. For example, you may want to apply the `truthy` function and a `length` function to the same property.
```
Comment (dotslashderek) [inline 2a.-style-guides/c-create-rules.md:53]: "apply multiple"

File: 7.-projects/working-with-files.md [hunk 3]
```diff
@@ -76,7 +76,11 @@ Adds a folder that you can use to organize content. This is especially useful fo
 
 ### Import Style Guide
 
-Use to add a Spectral ruleset to an API project. This action replaces all local rules, targets, and custom functions in an API project. It doesn't impact rules, targets, and functions that are inherited from an enabled style guide. See the [Style Guides documentation](../2a.-style-guides/a.style-guide-projects.md) for more information.
+Use to add a Spectral ruleset and custom functions to an API or style guide project. This action replaces all local rules, targets, and custom functions in an API project. It doesn't impact rules, targets, and functions that are inherited from an enabled style guide.
+
+You can import a single Spectral ruleset file or a `zip` file that includes a Spectral ruleset file and `.js` files that include custom functions.
```
Comment (dotslashderek) [inline 7.-projects/working-with-files.md:81]: Might be here or somewhere above, but I think we should mention that .js files containing custom functions are imported using the `functionsDir` property from the Spectral ruleset file (so if you have `functionsDir: ./custom_functions`, that's the only directory we look in to find those custom_functions - at least that's how it was working before, might have @P0lip or @mmiask confirm this is still the case

**Review by dotslashderek — COMMENTED** [review]
@pamgoodrich found a type, a couple questions / comments but looks really good!

File: 7.-projects/working-with-files.md (hunk 3 above)
Comment (P0lip) [reply 7.-projects/working-with-files.md:81]: Correct

Comment (pamgoodrich) [reply 7.-projects/working-with-files.md:81]: I think it's better to have this here:

https://meta.stoplight.io/docs/platform/branches/pam-441-multiple-thens/9spktiiuhvpbg-import-style-guides

I added this to the bottom of the "Import Guidelines" section:

The import process uses the [Spectral `functionsDir` keyword](https://meta.stoplight.io/docs/spectral/ZG9jOjI1MTkw-custom-functions#changing-directory) in the ruleset you are importing to locate custom functions. 

Does that work?

Comment (dotslashderek) [reply 7.-projects/working-with-files.md:81]: Yes I like that!

**Review by dotslashderek — APPROVED** [review]
Look's great Pam!

File: 2a.-style-guides/c-create-rules.md [hunk 4]
```diff
@@ -11,69 +11,55 @@ See these [rule examples](g-rule-examples.md) to get started.
...
 1. Edit one of these projects:
-   - A style guide project so the rule can be automatically applied to API projects that use that style guide (Professional and Enterprise plans only). 
+   - A style guide project so the rule can be automatically applied to API projects that use that style guide ([Professional and Enterprise plans only](https://stoplight.io/pricing)). 
    - An API project to add the rule to that project only. 
 2. Select the **Styles** tab.
-3. Select the **Add** icon (+), and then select **Add Rule**.
-
-![Add a Rule](https://stoplight.io/api/v1/projects/cHJqOjI/images/XmlF2nrOcSM)
-
+3. In the left pane, select the **Add (+)** icon next to **Rules**. (In a Style Guide project, select the **Add (+)** icon from project toolbar, and then select **Add Rule**.)
```
Comment (heitortsergent) [inline 2a.-style-guides/c-create-rules.md:17]: More of a style comment, it reads a little bit weird to me having the instruction and then another step in parenthesis like that, but that could also be just me!

I think I would just get rid of the parenthesis here, or break that into a sub-bullet.

File: 2a.-style-guides/h-create-custom-functions.md [hunk 5]
```diff
@@ -10,51 +10,16 @@ Once you've written custom functions, add them to your style guide and use them
...
 
-![Style Guide Custom Functions](https://stoplight.io/api/v1/projects/cHJqOjI/images/a1LibPsCCqc)
+![Style Guide Custom Functions](https://stoplight.io/api/v1/projects/cHJqOjI/images/wZwekKMKrbU)
 
 To create a custom function:
 
 1. Edit a style guide project or an API project that has a style guide enabled.
 2. Select the **Styles** tab.
-3. Select the **Add** icon (+), and then select **Add Function**.
+3. In the left pane, select the **Add (+)** icon next to **Functions**. (In a Style Guide project, select the **Add (+)** icon from project toolbar, and then select **Add Function**.)
```
Comment (heitortsergent) [inline 2a.-style-guides/h-create-custom-functions.md:19]: Same comment as above 🤓

File: 2a.-style-guides/c-create-rules.md [hunk 6]
```diff
@@ -11,69 +11,55 @@ See these [rule examples](g-rule-examples.md) to get started.
...
 
-- **Name**
+- **Name**: Provide a short descriptive name that contains no spaces. Example: `require-description`.
 
-   Provide a short descriptive name that contains no spaces. Example: `require-description`.
-
-- **Message**
-  
-   Provide the message that will appear in the **API Design Checker** as developers work with APIs. Add message text or include placeholders that are evaluated at runtime:
+- **Message**: Provide the message that will appear in the **API Design Checker** as developers work with APIs. Add message text or include placeholders that are evaluated at runtime:
```
Comment (heitortsergent) [inline 2a.-style-guides/c-create-rules.md:33]: Maybe replace "message text" with just "text" here.

File: 2a.-style-guides/c-create-rules.md [hunk 7]
```diff
@@ -11,69 +11,55 @@ See these [rule examples](g-rule-examples.md) to get started.
...
+- **Format**: Optionally set one or more formats for the rule. This is useful for overriding the global formats (OAS2 and OAS 3.x) for the rule.
 
 ### 2. Given Settings
 
-Set the area of the API specification the rule applies to.
-
-- **Target**
-
-  Select a target to specify the area of an API specification document. Targets are usually predefined and are required for each rule. Targets may be organized into [groups based on their names](b-create-targets.md#organize-targets). If you don't see a target you need, you can [add one here](b-create-targets.md). Note that inline targets can't be reused or start with the # symbol.
+Select a **target** to specify an area of an API specification document. Targets are usually predefined and are required for each rule. Targets may be organized into [groups based on their names](b-create-targets.md#organize-targets). If you don't see a target you need, you can [add one here](b-create-targets.md). Note that inline targets can't be reused or start with the # symbol.
```
Comment (heitortsergent) [inline 2a.-style-guides/c-create-rules.md:47]: I think saying "add one here" could be a little confusing, maybe saying "add one", or "create a new one"?

File: 2a.-style-guides/c-create-rules.md [hunk 8]
```diff
@@ -11,69 +11,55 @@ See these [rule examples](g-rule-examples.md) to get started.
...
+You can apply multiple **Then** settings per rule. This enables you to apply multiple functions to a rule rather than create a rule for each function you need. For example, you may want to apply the `truthy` function and a `length` function to the same property.
 
-### 3. Then Settings
+You must have at least one **Then** setting.
 
-Use Stoplight's [core functions](https://meta.stoplight.io/docs/spectral/ZG9jOjExNg-core-functions) to set the function that will evaluate your API design content. [Enable the Stoplight Style Guide](d-enable-style-guide.md) to use an additional set of custom functions.
+- **Property**: Add properties to reference a specific entity in the specified target. For example, when you target the `API_Tags` area of the OAS 3.1 specification, enter `description` to enforce rules on tag descriptions.
 
-Functions are listed in alphanumeric order.
+- **Function**: Select a function(required). You can:
```
Comment (heitortsergent) [inline 2a.-style-guides/c-create-rules.md:59]: Mhm, what does function(required) mean?

File: 2a.-style-guides/import-style-guides.md [hunk 9]
```diff
@@ -8,11 +8,19 @@ You can import [Spectral](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-
 
 This is useful when you have a ruleset that's maintained outside of Stoplight. When changes are made to a Spectral ruleset, you can reimport a new version of the file to replace the old ruleset.
 
-When you import Spectral files into a project, all local rules, targets, and custom functions are replaced. Inherited rules, targets, and functions aren't impacted. 
+## Import Guidelines
 
-To import a Spectral file:
+- When you import Spectral files into a project, all local rules, targets, and custom functions are replaced. Inherited rules, targets, and functions aren't impacted.
```
Comment (heitortsergent) [inline 2a.-style-guides/import-style-guides.md:13]: Can we highlight this part somehow? Maybe just using bold, or maybe using a warning tag?

File: 7.-projects/working-with-files.md [hunk 10]
```diff
@@ -76,7 +76,11 @@ Adds a folder that you can use to organize content. This is especially useful fo
 
 ### Import Style Guide
 
-Use to add a Spectral ruleset to an API project. This action replaces all local rules, targets, and custom functions in an API project. It doesn't impact rules, targets, and functions that are inherited from an enabled style guide. See the [Style Guides documentation](../2a.-style-guides/a.style-guide-projects.md) for more information.
+Use to add a Spectral ruleset and custom functions to an API or style guide project. This action replaces all local rules, targets, and custom functions in an API project. It doesn't impact rules, targets, and functions that are inherited from an enabled style guide.
```
Comment (heitortsergent) [inline 7.-projects/working-with-files.md:79]: Same comment as above, maybe we can highlight the part of "This action replaces all local rules, targets and custom functions in an API project."

**Review by heitortsergent — APPROVED** [review]
LGTM! Just left some small comments on style and highlighting some sections.

File: 2a.-style-guides/c-create-rules.md (hunk 4 above)
Comment (pamgoodrich) [reply 2a.-style-guides/c-create-rules.md:17]: Yeah... I realized that you can only use the + at the top in a style guide project. Darn it. I broke it into sub-bullets.

File: 2a.-style-guides/h-create-custom-functions.md (hunk 5 above)
Comment (pamgoodrich) [reply 2a.-style-guides/h-create-custom-functions.md:19]: Done

File: 2a.-style-guides/c-create-rules.md (hunk 6 above)
Comment (pamgoodrich) [reply 2a.-style-guides/c-create-rules.md:33]: Done

File: 2a.-style-guides/c-create-rules.md (hunk 7 above)
Comment (pamgoodrich) [reply 2a.-style-guides/c-create-rules.md:47]: Done

File: 2a.-style-guides/c-create-rules.md (hunk 8 above)
Comment (pamgoodrich) [reply 2a.-style-guides/c-create-rules.md:59]: Just missed the space. Functions are required.

File: 2a.-style-guides/import-style-guides.md (hunk 9 above)
Comment (pamgoodrich) [reply 2a.-style-guides/import-style-guides.md:13]: I think we're okay with what we have. You get a really big warning in the software when you do this.

**Pull request merged**